examples/finetune_resnet/config.jsonnet

```diff
@@ -0,0 +1,67 @@
+local data_dir = "./data/hymenoptera_data/";
```

Comment:
These examples should be self-contained. Is there a dataset on HuggingFace that we could use instead? Maybe one of these: https://huggingface.co/datasets?task_ids=task_ids:image-classification&sort=downloads